tox.ini

```diff
@@ -15,27 +18,23 @@ install_command = pip install -e ".[testing]" -U {opts} {packages}
 commands = coverage run testmanage.py test --deprecation all
 
 basepython =
-    python3.6: python3.6
     python3.7: python3.7
     python3.8: python3.8
+    python3.9: python3.9
+    python3.10: python3.10
 
 deps =
     coverage
 
-    django2.2: Django>=2.2,<2.3
-    django3.0: Django>=3.0,<3.1
-    django3.1: Django>=3.1,<3.2
     django3.2: Django>=3.2,<3.3
     django4.0: Django>=4.0,<4.1
+    django4.1: Django>=4.1,<4.2
     djangomaster: git+https://github.com/django/django.git@master#egg=Django
 
-    wagtail2.11: wagtail>=2.11,<2.12
-    wagtail2.12: wagtail>=2.12,<2.13
-    wagtail2.13: wagtail>=2.13,<2.14
-    wagtail2.14: wagtail>=2.14,<2.15
     wagtail2.15: wagtail>=2.15,<2.16
     wagtail2.16: wagtail>=2.16,<3.0
     wagtail3.0: wagtail>=3.0,<3.1
+    wagtail4.0: wagtail>=4.0,<4.1
```

Comment:
note: Wagtail 4.1 has since been released. As it is an LTS, it would be good to include it in the test matrix. Could be done in a follow up PR as long as it happens

Reply:
It will happen in the next round for 4.1 upgrades. We are tracking all packages now.

Reply:
Hi @zerolab , @nickmoreton,
I have also made a follow up PR for the added testing for wagtail 4.1 here:
https://github.com/torchbox/wagtail-content-import/pull/52

I just added a new envlist on the tox.ini file and included some tests in the test.yml
Thanks!